DMWM/activate-start-agent.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/sh
```

Comment:
You can add "-ex" to immediately terminate after a command finishes with an error, and to show the command that is being executed.